tools/workspace-plugin/src/plugins/clean-plugin.ts

```diff
@@ -63,3 +61,15 @@ function createNodesInternal(
     },
   };
 }
+
+export function buildCleanTarget(
+  _options: CleanPluginOptions,
```

Comment:
resolving unused lint errors by using `_` prefix. 

in order to have uniform API this is the tax we have to pay for now  - we don't use those args ATM within this builder.